Lib/test/test_fnmatch.py

```diff
@@ -152,6 +152,12 @@ def test_filter(self):
         self.assertEqual(filter([b'Python', b'Ruby', b'Perl', b'Tcl'], b'P*'),
                          [b'Python', b'Perl'])
 
+    def test_filter_multipattern(self):
+        self.assertEqual(filter(['Python', 'Ruby', 'Perl', 'Tcl'], ('P*', 'T*')),
+                         ['Python', 'Perl', 'Tcl'])
+        self.assertEqual(filter([b'Python', b'Ruby', b'Perl', b'Tcl'], (b'P*', b'T*')),
```

Comment:
The PEP 8 style guide states that lines should be limited to 79 characters.

Reply:
Addressed your comments on Dec 13, 2020 and forgot to tell you :sweat_smile: